Makefile

```diff
@@ -97,38 +97,39 @@ analyse_canary: ## run php-cs-fixer and phpstan (dark-canary)
 ##@ DEPLOY
 
 ACCOUNT=$(shell aws sts get-caller-identity --profile=poser | jq -r '.Account')
-VER=$(shell date +%s)
+AWS_PROFILE ?= poser
 
 deploy_prod: ## deploy to prod
-	# docker_image_phpfpm
+	aws ecr get-login-password --profile ${AWS_PROFILE} | docker login --password-stdin -u AWS $(ACCOUNT).dkr.ecr.eu-west-1.amazonaws.com; \
```

Comment:
don't kill me .. why curly brackets here (I love them, do not get me wrong :D .. If I'd have to pick a choice I'd use curly brackets everywhere for variables in fact)?

Reply:
removed curly bracket